scripts/linters/pylint_extensions_test.py

```diff
@@ -3257,3 +3257,70 @@ def some_random_function(param1, param2):
 
         with self.checker_test_object.assertNoMessages():
             self.checker_test_object.checker.visit_functiondef(def_node)
+
+
+class NoBlankLineBelowFunctionDefinitionTests(unittest.TestCase):
+
+    def setUp(self):
+        super(NoBlankLineBelowFunctionDefinitionTests, self).setUp()
+        self.checker_test_object = testutils.CheckerTestCase()
+        self.checker_test_object.CHECKER_CLASS = (
+            pylint_extensions.NoBlankLineBelowFunctionDefinition)
+        self.checker_test_object.setup_method()
+
+    def test_empty_line_below_function_definition(self):
```

Comment:
The 2 tests should be like:
1. `test_empty_line_below_function_definition_raises_error` - Assert that the error message is added.
2. `test_no_empty_line_below_function_definition_raises_no_error ` - Assert that no message is raised.
You don't need to add any new message for the test 2.

Reply:
Corrected sir!

Reply:
Also, thanks for the helpful guide link!